manifest/docker_schema2.go

```diff
@@ -122,6 +122,8 @@ type Schema2V1Image struct {
 	Architecture string `json:"architecture,omitempty"`
 	// OS is the operating system used to build and run the image
 	OS string `json:"os,omitempty"`
+	// Variant is a variant of the CPU that the image is build and runs on
+	Variant string `json:"variant,omitempty"`
```

Comment:
In all three places, please move this just below `Architecture`
- That is consistent with docker/docker
- It is a variant of the architecture, not of the OS.